CONTRIBUTING.md

```diff
@@ -119,7 +119,32 @@ Availability  Capabilities  CapabilityDescriptions
 
 #### Networking issues
 
-If the issue is about networking, run [networking.bat](https://github.com/Microsoft/WSL/blob/master/diagnostics/networking.bat) in an administrative command prompt:
+Install tcpdump in your WSL distribution using the following commands.
+Note: This might not work if WSL has Internet connectivity issues.
+
+```
+# sudo apt-get update
+# sudo apt-get -y install tcpdump
+```
+
+Run [networking_diagnostics.bat](https://github.com/Microsoft/WSL/blob/master/diagnostics/networking_diagnostics.bat) in an administrative command prompt:
+
+```
+$ git clone https://github.com/microsoft/WSL --depth=1 %tmp%\WSL
```

Comment:
The git command might not be available on the user's machine so I'd recommend downloading the files with powershell instead

Reply:
fixed in iteration 3